src/v2/Components/AstroChat/useAstro.ts

```diff
@@ -75,13 +75,13 @@ const loadMessage = async (
 
       if (isCommandType(resolvedContent)) {
         message.command = {
-          type: CommandType.COMMAND,
+          type: resolvedContent.command as CommandType,
           params: {
             args: resolvedContent.args ?? [],
             channels: resolvedContent.channels,
             command: resolvedContent.command,
           },
-        };
+        } as Command;
```

Comment:
Could you check why these type 2 assertions are required?  (Command and CommandType)

Reply:
Double check me, but from the v2 conversion we ended up with an extra ResponseCommand class (like the one in postTalk) that extended the BaseCommand type. Removing that fixed the issue.